doc/languages-frameworks/vim.section.md

```diff
@@ -243,10 +231,23 @@ Alternatively, set the number of processes to a lower count to avoid rate-limiti
 ./pkgs/applications/editors/vim/plugins/update.py --proc 1
 ```
 
-## Important repositories {#important-repositories}
+## How to maintain an out-of-tree overlay of vim plugins ?
+
+You can use the updater script to generate basic packages out of a custom vim
+plugin list:
+```
+pkgs/applications/editors/vim/plugins/update.py -i vim-plugin-names -o generated.nix --no-commit
```

Comment:
Can you run this out of tree, i.e. with `nix-shell`?

Reply:
That would be a logical followup indeed. Not sure how to expose that yet so I'll postponethe suggestion.